langgraph/demo.py

```
@@ -0,0 +1,151 @@
+
+@tool
+def mcp_file_summary(filename: str, projectname: str):
+    """
+    Use this when trying to build more context for any repository context you need. 
+    Pass parameters filename=name on the file and projectname=name of the target project in original user ticket or prompt
+    Returns the contents of the file from this project.
+    """
+    try:
+        # Run the async function in a new event loop
+        response = asyncio.run(read_project_file_mcp(file_name=filename, project_name=projectname))
+        return f"The file contents of {filename} from {projectname}: {response}"
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)